### ISSUE TITLE
In case empty line in test starts with tabulator it is propagated to table view wrongly

### ISSUE THREAD

**Issue by adam-wypych** [nokia/RED#145]
Red standalone version: 0.8 (windows).

Below test file ( `test1.robot` ):

```robotframework
*** Test Cases ***
Test1
       Log    theFirstLine
       
|     |    |

     Log   theLastLine
```

The output in table view looks like below:
![image](https://user-images.githubusercontent.com/12131969/32057283-ae9ecd06-ba5e-11e7-836f-53bab3a92029.png). 
Note: letter `d` was added just to show what happens.
Additionally the pipe line (`|`) is totally omitted in the view.

**Comment by adam-wypych** [comment]
Can you confirm that this is the bug?

**Comment by michalanglart** [comment]
Fixed in 0.8.12
